docs/source/cuda/overview.rst

```diff
@@ -60,9 +60,15 @@ Presently 11.2 is the minimum required toolkit version. An NVIDIA driver
 sufficient for the toolkit version is also required (see also
 :ref:`minor-version-compatibility`).
 
-If you are using Conda, you can install the CUDA toolkit with::
+Conda users can install the CUDA Toolkit into a conda environment.
 
-   $ conda install cudatoolkit
+For CUDA 12, ``cuda-nvcc`` and ``cuda-nvrtc`` are required::
+
+    $ conda install -c conda-forge cuda-nvcc cuda-nvrtc "cuda-version>=12.0"
+
+For CUDA 11, ``cudatoolkit`` is required::
+
+    $ conda install -c conda-forge cudatoolkit "cuda-version=11.*"
```

Comment:
Maybe we can drop the `*` from this and similar lines

It is effectively implied. Also `*` sometimes confuses shells

```suggestion
    $ conda install -c conda-forge cudatoolkit "cuda-version=11"
```